packages/types/lib/environment/db.ts

```diff
@@ -39,6 +39,7 @@ export interface DBEnvironment extends Timestamps {
     slack_notifications: boolean;
 
     webhook_receive_url?: string;
+    otlp_settings?: { endpoint: string; headers: Record<string, string> } | null;
```

Comment:
remember to remove `?` on the next PR 👌🏻

Reply:
thanks as you saw I had forgotten